cap-async-std/src/fs_utf8/dir_builder.rs

```diff
@@ -51,7 +51,7 @@ impl DirBuilder {
 #[cfg(unix)]
 impl async_std::os::unix::fs::DirBuilderExt for DirBuilder {
     #[inline]
-    fn mode(&mut self, mode: libc::mode_t) -> &mut Self {
+    fn mode(&mut self, mode: u32) -> &mut Self {
```

Comment:
Now that I started porting to Windows, this is actually a welcome change since we probably won't pull in `libc` when targeting Windows.